projectDocs/testing/automated.md

```diff
@@ -50,3 +50,10 @@ To run all tests standard tests for developers use `runsystemtests.bat --include
 Internally this script uses the Robot test framework to execute the tests.
 Any arguments given to `runsystemtests.bat` are forwarded onto Robot.
 For more details (including filtering and exclusion of tests) see `tests/system/readme.md`.
+
+### License checks
+
+NVDA uses GPLv2 which is incompatible with certain licenses like Apache.
```

Comment:
is it "v2" or "v2 or later"

Reply:
our current license is just GPLv2 with 2 modification
https://github.com/nvaccess/nvda/blob/8fb8ffcaba3950b1e4ca7b58e165267fc5369eb9/copying.txt